internal/server/server.go

```diff
@@ -264,21 +282,26 @@ func getGraphRead(graphDB knowledge.GraphDB) http.HandlerFunc {
 			return
 		}
 
-		gJson, err := json.Marshal(g)
+		gJSON, err := json.Marshal(g)
 		if err != nil {
 			replyWithInternalError(w, err)
 			return
 		}
 
-		if _, err := w.Write(gJson); err != nil {
+		if _, err := w.Write(gJSON); err != nil {
 			replyWithInternalError(w, err)
 		}
 	}
 }
 
-func postGraphUpdates(graphUpdatesC chan knowledge.SourceSubGraphUpdates) http.HandlerFunc {
+func postGraphUpdates(registry importers.Registry, graphUpdatesC chan knowledge.SourceSubGraphUpdates) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
-		ok, source := isTokenValid(r)
+		ok, source, err := isTokenValid(registry, r)
+		if err != nil {
+			replyWithInternalError(w, err)
```

Comment:
err means missing token => should return a 401 (or HTTP 400) instead of a HTTP 500